src/Terrajobst.PlatformCompat.Analyzers.Tests/DeprecatedAnalyzerTests.cs

```diff
@@ -189,5 +189,32 @@ static void Main()
 
             AssertMatch(source, expected);
         }
+
+        [Theory]
+        [InlineData("Win32S")]
+        [InlineData("Win32Windows")]
+        [InlineData("WinCE")]
+        [InlineData("Xbox")]
+        [InlineData("MacOSX")]
+        public void DeprecatedAnalyzer_Triggers_DE0007(string memberName)
+        {
+            var source = @"
+                using System;
+
+                class Program
+                {
+                    static void Main()
+                    {
+                        var x = PlatformID.{{MEMBER_NAME}} x = null;
```

Comment:
Maybe I'm missing something, but this line doesn't seem like valid C# with the extra `x = null` at the end.